pkg/security/probe/consts.go

```diff
@@ -370,6 +370,9 @@ func bitmaskToString(bitmask int, intToStrMap map[int]string) string {
 type OpenFlags int
 
 func (f OpenFlags) String() string {
+	if int(f) == syscall.O_RDONLY {
```

Comment:
How can you know if it's O_RDONLY or O_LARGEFILE?

Reply:
Can O_RDONLY be used with other flags? A quick look at https://pubs.opengroup.org/onlinepubs/007908799/xsh/open.html seems to indicate it's possible (eg. O_NONBLOCK). In this case, we would only return "O_RDONLY"?

Reply:
you can't, but O_LARGEFILE is supposed to be set internally by the libc while doing syscall. So we can set the value by hand no using the golang definition but I don't see any use cases for it

Reply:
According to https://stackoverflow.com/questions/37891184/checking-only-the-o-rdonly-flag-to-open2, O_RDONLY/O_WRONLY/O_RDWR need some special care, maybe worth doing it?